lib/packages/docutils/rstgen.nim

```diff
@@ -755,7 +757,12 @@ proc renderHeadline(d: PDoc, n: PRstNode, result: var string) =
 
   # Generate index entry using spaces to indicate TOC level for the output HTML.
   assert n.level >= 0
-  setIndexTerm(d, changeFileExt(extractFilename(d.filename), HtmlExt), refname, tmp.stripTocHtml,
+  let
+    htmlFileRelPath = if d.destBaseDir != "".AbsoluteDir: # d is initialized in docgen
```

Comment:
for next time (or in cleanup PR) you can use `isEmpty`

Reply:
Thanks, I have force-pushed an edit with your suggestion.